### DISCUSSION TITLE
Why serenity used <div> instead of <button> for tool buttons?

### DISCUSSION THREAD

**Discussion by dfaruque** [serenity-is/Serenity#6963] [General]
Currently, serenity is rendering a button using `<div>`. which looks like the following

```
<div class="tool-button add-button icon-tool-button">
    <div class="button-outer">
        <span class="button-inner"><i class="fa fa-plus-circle text-green"></i> New Product</span>
    </div>
</div>
```
but the same can be achieved using `<button>`

`<button class="add-button"><i class="fa fa-plus-circle"></i> New Product</button>`

Is there any reason for using `<div>` instead of `<button>`?

**Comment by volkanceylan** [comment]
Once upon a time, we were using Flexigrid, jquery-ui and its toolbar markup, which had that 3 div inside each other for styling. Probably they chose that as toolbuttons were not supposed to be buttons, and styling browser elements were not that easy, not sure. Now if we change the markup, it would be a breaking change.

**Reply by dfaruque** [reply]
Thank you for the reply.
